src/pages/product/[id].tsx

```diff
@@ -0,0 +1,59 @@
+import { GetServerSideProps } from 'next';
+import Image from 'next/image';
+
+const URL = process.env.API_URL_SERVER;
+
+interface ProductType {
+  id: number;
+  imageURL: string;
+  productName: string;
+  description: string;
+  price: number;
+}
+
+export default function ProductDetailPage({
+  product,
+}: {
+  product: ProductType;
+}) {
+  return (
+    <div className="mx-auto flex w-[80%] flex-col gap-5 pt-10 md:flex md:w-[90%] md:flex-row md:gap-15">
```

Comment:
nit: 이렇게 표기할 수 있습니다.
```suggestion
    <div className="mx-auto flex w-4/5 flex-col gap-5 pt-10 md:flex md:w-9/10 md:flex-row md:gap-15">
```

Reply:
000d436057e8de60d89748cceade537a216e4f88

수정했습니다~!!